INSTALLATION.md

```diff
@@ -0,0 +1,151 @@
+# Intstallation Guide
+
+## Prerequisites
+
+### Check if OpenStack system is functional
+* Build, destroy VMs, etc.
+
+### Steps to prepare environment (on each host)
+
+1. Verify VLan is connected, e.g. by pinging from one infra host to the other
+2. Setup Bridge and v-eth pair (see ![Alt](rpc-octavia-network-diagram.png "RPC Octavia Network Diagram")) The VLAN_ID can be adjusted to the local circumstances. This script will set up the br-lbaas script and create a link between br-lbaas and br-vlan in a post-up script. This ensures the link survives a reboot.
```

Comment:
Let's try to use markdown to clearly articulate parameters, i.e. `VLAN_ID` instead of just VLAN_ID.

Reply:
Also I don't see anything here about any need to adjust `address` as a function of the environment.